mypy/git.py

```diff
@@ -1,18 +1,28 @@
 """Utilities for verifying git integrity."""
 
+from __future__ import print_function
+
 # Used also from setup.py, so don't pull in anything additional here (like mypy or typing):
 import os
 import pipes
 import subprocess
 import sys
 
 
-def is_git_repo(dir: str) -> bool:
+# NOTE: Cannot use typing.TYPE_CHECKING guard here because cannot import
+#       "typing" in this file at all.
+if False:
+    from typing import Iterator
```

Comment:
I prefer this:
```
MYPY = False
if MYPY:
    from typing import Iterator
```
We'll have to support MYPY basically forever, while "if False" might eventually cause the body to be considered unreachable.